docs/_pages/introduction.md

```diff
@@ -162,13 +170,39 @@ The above will batch the two failures, and throw an exception at the point of di
 E.g. Exception thrown at point of dispose contains:
 
 ```text
-Expected value to be 10, but found 5.
+Expected value to be 10, but found 5 (difference of -5).
```

Comment:
I think if the difference is less than or equal to ABS(10) no difference is shown (at least for non-floating point types).

Reply:
Yeah, you're right. Thanks.

Reply:
Am I missing something? 100 - 95 is also difference < ABS(10) 🤔